Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/person/core/PersonCertificatesSettings.kt

```diff
@@ -77,6 +77,33 @@ class PersonCertificatesSettings @Inject constructor(
         }
     }
 
+    suspend fun setGStatusNotifiedAt(
```

Comment:
Tests

Reply:
Yes, tests are comming, that's why the PR is in draft mode for now and I've included the TODO checkboxes in the description.